tools/ci_build/build.py

```diff
@@ -1533,8 +1533,7 @@ def generate_build_tree(
                 ldflags = ["/profile", "/DYNAMICBASE"]
                 # Address Sanitizer libs do not have a Qspectre version. So they two cannot be both enabled.
                 if not args.enable_address_sanitizer:
-                    # Also enable a special perf patch that was made for Intel Meteor Lake mobile CPUs
-                    cflags += ["/Qspectre", "/DONNXRUNTIME_ENABLE_INTEL_METEOR_LAKE_MOBILE_PLATFORM_PERF_PATCH"]
```

Comment:
This flag is default ON. It is enabled in every ONNX Runtime's official package.  It was added because if in any case the code path doesn't work, at least we have a fallback. I can tell the customer to build the code without this flag.
Is there any benefit if we remove it?